src/App.js

```diff
@@ -1,10 +1,51 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Header from './components/Header';
+import Editor from './Editor';
+import { Badge } from './components/Badge';
+
+const INITIAL_DATA = {
+  time: new Date().getTime(),
+  blocks: [
+    {
+      type: 'paragraph',
+      data: {
+        text: 'A long time ago',
+        level: 1,
+      },
+    },
+  ],
+};
 
 function App() {
+  const [data, setData] = useState(INITIAL_DATA);
+  const [titleName, setTitle] = useState('Title Name');
+  const [badge, setBadge] = useState('Book');
+  const [chapters, setChapters] = useState(['Chapter Name']);
+
+  const requestData = () => {
+    setData(INITIAL_DATA);
+    setTitle('Title Name');
+    setChapters(['Chapter Name']);
+    setBadge('Book');
+  }
+
+  useEffect(()=>{
+    requestData();
+  },[]);
+
   return (
     <React.Fragment>
-      <Header />
+      <Header>
+        <div>
+          {titleName} <Badge size="xs">{badge}</Badge>
+        </div>
```

Comment:
1. On the second thought. Let's move it from the header to the body as the designers did. (sorry, changed my mind :) )
2. I believe there should not be just text in html. Let's use `span` at least. wdyt?